marketplace/netlify-build/extension.json

```diff
@@ -0,0 +1,53 @@
+{
+  "id": "netlify-button",
+  "name": "Netlify Build and Preview",
+  "src": "http://localhost:1234",
+  "sidebar": true,
+  "parameters": {
+    "installation": [
+      {
+        "id": "publishKey",
+        "type": "Symbol",
+        "name": "PubNub publish key",
+        "description": "A public PubNub publish key to publish update events to Netlify",
+        "required": true
+      },
+      {
+        "id": "subscribeKey",
+        "type": "Symbol",
+        "name": "PubNub subscribe key",
+        "description": "A public PubNub subscribe key to listen for update events from Netlify",
+        "required": true
+      },
+      {
+        "id": "channels",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify channel list (Max 3)",
+        "description": "Example: my-app-abc,my-second-app-def,my-third-app-ghi"
+      },
+      {
+        "id": "netlifySiteIds",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify Id list (Max 3)",
+        "description": "Example: 14234e3,143248,35233"
+      },
+      {
+        "id": "buildHookUrls",
+        "type": "Symbol",
+        "name": "Comma seperated Netlify build hook URLs (Max 3)",
+        "description": "https://api.netlify.com/build_hooks/{site-id}"
```

Comment:
"Example: https://api.netlify.com/build_hooks/xyz,https://api.netlify.com/build_hooks/abc". There are no site IDs in URLs.